thoth/messaging/unresolved_package.py

```diff
@@ -39,9 +41,9 @@ class MessageContents(faust.Record, serializer="json"):
         """Class used to represent contents of a message Kafka topic."""
 
         package_name: str
-        package_version: str
-        index_url: str
-        runtime_environment: str
```

Comment:
why is this removed?

Reply:
better to keep it general? I extracted the information needed for solver from adviser report part related to runtime environment and built the solver (string) that need to be scheduled with priority, if no runtime environment found, all solvers will be scheduled for that package.

```
{'cuda_version': None, 'hardware': {'cpu_family': None, 'cpu_model': None}, 'name': 'rhel:8', 'operating_system': {'name': 'rhel', 'version': '8.0'}, 'python_version': '3.6'}
```

`solver=solver-rhel-8.0-py36`

Reply:
WDYT @goern ?